lisa/sut_orchestrator/azure/common.py

```diff
@@ -99,6 +99,7 @@ class NodeContext:
     username: str = ""
     password: str = ""
     private_key_file: str = ""
+    use_public_ip_address: bool = True
```

Comment:
Make names consistent like `use_public_ip_address` on other places too.

Reply:
ok ... was trying to match the collocated params (it is public_ip_address here, public_address in other places).

Reply:
in some places there is a public_address property ... and others there is a public_ip_address property.

i was trying to match whatever was local (use_public_address when public_address is applicable, use_public_ip_address when public_ip_address is applicable).  if i'm not doing that, which do you prefer:  use_public_address or use_public_ip_address?

Reply:
if so, `use_public_address` is shorter and better. :)